src/System.Net.Http.WinHttpHandler/src/System/Net/Http/WinHttpResponseParser.cs

```diff
@@ -17,6 +17,7 @@ internal static class WinHttpResponseParser
         private const string EncodingNameDeflate = "DEFLATE";
         private const string EncodingNameGzip = "GZIP";
 #if NET46
+        private static readonly Version HttpVersion20 = new Version(2,0);
```

Comment:
One of these was already defined/allocated.  Can it just be defined somewhere once?